hathor/transaction/static_metadata.py

```diff
@@ -56,9 +71,183 @@ def from_bytes(cls, data: bytes, *, target: 'BaseTransaction') -> 'VertexStaticM
 @dataclass(slots=True, frozen=True, kw_only=True)
 class BlockStaticMetadata(VertexStaticMetadata):
     height: int
+
+    # A list of feature activation bit counts. Must only be used by Blocks, is None otherwise.
```

Comment:
This static metadata is for blocks only, right? I mean, I guess it will never be None.

Reply:
Yes, it was outdated. Changed in https://github.com/HathorNetwork/hathor-core/pull/1014/commits/86250107215aa73a661e4437d5c9287ec2f5ba39

Reply:
Thanks!